conversational-skills/procode-skill-sdk-js/src/mock-server/app.js

```diff
@@ -0,0 +1,75 @@
+// import dependencies and initialize express
+
+import express from 'express';
+import bodyParser from 'body-parser';
+
+import {createBluepointsSkill} from '../bluepoints/BluepointsSkillFactory.js';
+
+const app = express();
+
+
+
+// enable parsing of http request body
+app.use(bodyParser.urlencoded({extended: true}));
+app.use(bodyParser.json());
+
+
+// routes and api calls
+
+const bluepointsSkill = await createBluepointsSkill('en'); 
+
+/**
+ * This function lists the skills available by your provider. It will be called when you select which skills are part of your Assistant.
```

Comment:
```suggestion
 * This function lists the skills available by your provider. It will be called by the Assistant UI when selecting skills to create a code based action.
```